### PR TITLE
US44

### PR DESCRIPTION
**Pull request by jcdesimp**
- [ ] Joe
- [x] James

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,72 +1,73 @@
...
+                loader: 'url-loader?limit=8192'
+            },
+            { 
+                test: /\.(json)$/,
+                loader: 'json-loader'
+            }
+        ]
+    },
+    plugins: [
+        // new webpack.optimize.UglifyJsPlugin({
```
Comment (JDonut) [inline webpack.config.js]: Do we need this commented stuff? If so, please add a note explaining what it's for.

File: src/js/components/userProfilePanel/connectedUserProfilePanel.js [hunk 2]
```diff
@@ -3,21 +3,18 @@
...
-        let decodedJWT = jwtdecode(state.authentication.jwt);
-        return {
-            name: decodedJWT.user_name,
-            orcid: decodedJWT.user_orcid_id
-        };
-    },
+    state => ({
+        name: state.userInfo.user_name,
+        orcid: state.userInfo.user_orcid_id,
+        email: state.userInfo.user_email
```
Comment (Laludia) [inline src/js/components/userProfilePanel/connectedUserProfilePanel.js]: Should the user's role:[] be here as well? I was thinking at the profile they should at least get info of what roles they're in.

File: src/js/components/loaders/dnaLoader.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
+"use strict";
```
Comment (Laludia) [inline src/js/components/loaders/dnaLoader.jsx]: Is this component meant for importing a gene entry? Not really sure what this is suppose to do atm but it looks like a placeholder.

File: src/js/components/userProfilePanel/connectedUserProfilePanel.js (hunk 2 above)
Comment (jcdesimp) [inline src/js/components/userProfilePanel/connectedUserProfilePanel.js]: @Laludia maybe eventually but for now this meets the acceptance criteria for the story.

File: src/js/components/loaders/dnaLoader.jsx (hunk 3 above)
Comment (jcdesimp) [inline src/js/components/loaders/dnaLoader.jsx]: @Laludia its a component that I want to eventually display a loading animation. Currently it just has the string "Loading...". The idea was to have it be like an animation of a DNA strand hence the name, i will rename it so it's more clear though.

Comment (Laludia) [inline src/js/components/loaders/dnaLoader.jsx]: Maybe you can name it loadingIcon.jsx

**Pull request merged**